### PR TITLE
refactor(generate_assets & generate_static_assets)

### PR DESCRIPTION
**Pull request by NekoMaru76**

### CODE REVIEW

**Comment by NekoMaru76** [comment]
Ignore the other 8 commits. I accidentally messed up a bit. Newbie mistakes at using GitHub.

**Comment by NekoMaru76** [comment]
Though files in JavaScript is quite weird. Where do we even specify `webkitRelativePath` into the `File` constructor and why does it even exist in first place? Do we need to manually modify it if we need to set the value? But if we tried to just `Object.assign` + `Blob` constructor without it, TS will be mad at me.

File: serve_test.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { assertStringIncludes } from "https://deno.land/std@0.155.0/testing/asserts.ts";
```
Comment (kt3k) [inline serve_test.ts:1]: Can you import this from `test_deps.ts`?

**Review by kt3k — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**